.github/workflows/web_generator.yaml

```diff
@@ -0,0 +1,67 @@
+name: package:web_generator
+permissions: read-all
+
+on:
+  # Run CI on pushes to the main branch and on PRs.
+  push:
+    branches: [ main ]
+    paths:
+      - '.github/workflows/web_generator.yaml'
+      - 'web_generator/**'
+  pull_request:
+    paths:
+      - '.github/workflows/web_generator.yaml'
+      - 'web_generator/**'
+  schedule:
+    - cron: "0 0 * * 0"
+
+defaults:
+  run:
+    working-directory: web_generator/
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [stable]
+        test_config: ['', '-p chrome', '-p chrome -c dart2wasm']
+
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
+        with:
+          sdk: ${{ matrix.sdk }}
+
+      - run: dart pub get
+      - run: dart format --output=none --set-exit-if-changed .
+        if: ${{ matrix.sdk == 'stable' }}
```

Comment:
We were formatting with `dev` before so I'd just remove.

Reply:
(actually, I removed - as you said, we're only running this workflow on `dev`)